CHANGELOG.rst

```diff
@@ -6,7 +6,20 @@ Changelog
 Unreleased
```

Comment:
Rm?

Reply:
I left that there for our next release, but yes, I can drop it until we have something to put there.

Reply:
Only because it's gonna be visible in the published docs.